src/MacInstaller/index.scss

```diff
@@ -0,0 +1,35 @@
+@use "sass:color";
+@use "sass:map";
+
+@use "../sass/tokens";
+@use "../sass/mixins";
+
+.mac-installer {
+  display: flex;
+  width: 100%;
+
+  align-items: center;
+
+  background-color: map.get(tokens.$brand, black);
+  border: 1px solid map.get(tokens.$brand, purple);
+  color: map.get(tokens.$brand, white);
+  text-decoration: none;
+
+  padding: 1rem;
```

Comment:
Would like to variable-ize stuff here as well eventually but this is fine for now

Reply:
Yeah, there are a ton of magic numbers I threw in to make it "just work" that we need to factor out.